spec/controllers/discovery_controller_spec.rb

```diff
@@ -54,7 +54,8 @@
       Doorkeeper::OpenidConnect.configure do
         protocol { :testing }
       end
-
+      pending('TODO: figure out if we want to support custom protocols here. This regression
+        comes from the fixes to the non-production http/https hardcoded protocols.')
```

Comment:
yeah, but it is something in the base library, so if we ever want to contribute back, we'll have to fix this bit. But I'll chalk that up to another day.